src/vulnerability.ts

```diff
@@ -12,6 +12,7 @@ const AnalyticsSource = "\nDependency Analytics Plugin [Powered by Snyk]";
 class Vulnerability {
     ecosystem: string;
     name: string;
+    parent: string;
```

Comment:
nit: Should this be called as module/mod instead of parent?

Reply:
Comment taken, changing it to 'module'